settings.py

```diff
@@ -2256,6 +2256,16 @@ def _determine_couch_databases(couch_databases):
     'test-pna': 'custom.yeksi_naa_reports',
 }
 
+OTHER_DOMAINS_WITH_CUSTOM_MODULES = [
+    'fri',
```

Comment:
it's ok to migrate this one, we're actually going to be removing the custom code from the codebase soon as well